docs/swagger.yml

```diff
@@ -918,7 +916,7 @@ paths:
         200:
           description: "Detailed report successfully retrieved"
           schema:
-            $ref: "#/definitions/FactCollectionOut"
+            $ref: "#/definitions/DetailsReport"
```

Comment:
Change this to DetailsReportOut possibly to keep the old structure of how FactCollection was